### ISSUE TITLE
Hint update about temple: dotnet new -i xxxx raise warning about netcoreapp1.0

### ISSUE THREAD

**Issue by tebeco** [dotnet/templating#2744]
Hello,

## Context
When attempting to do `dotnet new web|gitignore|foo|bar` the SDK hinted me about a possible update of the template from 5.0.0 to 5.0.1

```
> dotnet new gitignore
The template "dotnet gitignore file" was created successfully.

An update for template pack Microsoft.DotNet.Common.ItemTemplates::5.0.0 is available.
    install command: dotnet new -i Microsoft.DotNet.Common.ItemTemplates::5.0.1
```

## Expected
this should probably work and not mention `netcoreapp1.0`
```
> dotnet new -i Microsoft.DotNet.Web.ProjectTemplates.5.0::5.0.1
> dotnet new -i Microsoft.DotNet.Common.ItemTemplates::5.0.1
```

## The issue
Why does it mention `netcoreapp1.0` ?
The update seems to work though

![image](https://user-images.githubusercontent.com/2266487/104073652-889de100-520e-11eb-95f5-7bd24811ae72.png)

```
dotnet new -i Microsoft.DotNet.Web.ProjectTemplates.5.0::5.0.1
  Determining projects to restore...
C:\Program Files\dotnet\sdk\5.0.200-preview.20601.7\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.EolTargetFrameworks.targets(28,5): warning NETSDK1138: The target framework 'netcoreapp1.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy. [C:\Users\xxxx\.templateengine\dotnetcli\v5.0.200-preview.20601.7\scratch\restore.csproj]
  Restored C:\Users\xxxx\.templateengine\dotnetcli\v5.0.200-preview.20601.7\scratch\restore.csproj (in 1.93 sec).

Templates                                         Short Name               Language          Tags
--------------------------------------------      -------------------      ------------      ----------------------
Co.....
...
...
```

## details
`dotnet --info` :
```
 dotnet --info
.NET SDK (reflecting any global.json):
 Version:   5.0.200-preview.20601.7
 Commit:    b3b934bbf2

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.21286
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.200-preview.20601.7\

Host (useful for support):
  Version: 5.0.0
  Commit:  cf258a14b7

.NET SDKs installed:
  3.1.403 [C:\Program Files\dotnet\sdk]
  5.0.200-preview.20601.7 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.23 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.9 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Comment by vlada-shubina** [comment]
@tebeco 
We have opened the https://github.com/dotnet/templating/issues/2739 to disable update checks if needed and considering not publishing template package in case there is no change to templates. 

As for "netcoreapp1.0" issue, it was already fixed in 5.0.1 SDK, please see https://github.com/dotnet/templating/issues/2627. Please upgrade to SDK 5.0.1. 
Do not hesitate to reopen the issue if you still encounter it in 5.0.1. Thanks.

**Comment by tebeco** [comment]
hello @vlada-shubina 

thx a lot for the quick answer.
Auto discovery of update is fine ;)

I was more surprised about nc1.0, and yes apparently i forgot to update my sdk

thx
